include/compatibility.json

```diff
@@ -1,51 +1,56 @@
     {"compatible" : 1, "os" : "Linux", "osVersion" : "", "browser" : "Firefox", "versions" : [31]},
 
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "10", "browser" : "Firefox", "versions" : [40, 41, 42, 43, 44, 45, 50]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "10", "browser" : "Chrome", "versions" : [45, 46, 47, 48, 55]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "10", "browser" : "Firefox", "versions" : [40]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "10", "browser" : "Chrome", "versions" : [45]},
     {"compatible" : 1, "os" : "Windows", "osVersion" : "10", "browser" : "Internet Explorer", "versions" : [11]},
     {"compatible" : 1, "os" : "Windows", "osVersion" : "10", "browser" : "Edge", "versions" : []},
 
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "8.1", "browser" : "Firefox", "versions" : [27, 28, 29, 30, 36, 40, 41, 42, 43, 44, 45, 50]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "8.1", "browser" : "Chrome", "versions" : [33, 34, 35, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 55]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "8.1", "browser" : "Internet Explorer", "versions" : [10,11]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8.1", "browser" : "Firefox", "versions" : [27, 28, 29, 30, 36, 40]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8.1", "browser" : "Chrome", "versions" : [33, 34, 35, 40]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8.1", "browser" : "Internet Explorer", "versions" : [10]},
 
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Firefox", "versions" : [35,36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 50]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Chrome", "versions" : [39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 55]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Internet Explorer", "versions" : [10,11]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Firefox", "versions" : [35]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Chrome", "versions" : [39]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "8", "browser" : "Internet Explorer", "versions" : [10]},
 
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Firefox", "versions" : [34,35,36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 50]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Internet Explorer", "versions" : [10, 11]},
-    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Chrome", "versions" : [40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 55]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Firefox", "versions" : [34]},
+    {"compatible" : 1, "os" : "Windows", "osVersion" : "7", "browser" : "Chrome", "versions" : [40]},
```

Comment:
This is to allow for them to be tested. This question should be readdressed after qa has been able to test the different environments.